util/dvsim/sim_utils.py

```diff
@@ -99,3 +100,96 @@ def vcs_cov_summary_table(buf):
 
     # If we reached here, then we were unable to extract the coverage.
     raise SyntaxError(f"Coverage data not found in {buf.name}!")
+
+
+def get_runtimes(run_log_txt: str, tool: str) -> Tuple[int, int]:
+    """Capture and return two runtimes: job_runtime and simulated_time.
+
+    The run.log file is passed as an input to the function, in addition to the
+    tool used. The tool returns two integers if the runtimes were extracted
+    successfully. The function will raise appropriate exception if the extraction
+    fails.
+    Note that the file is parsed in a reversed order so that we can get the
+    simulated time from Xcelium log file.
+    """
+    with open(run_log_txt, 'r') as f:
+        lines = f.readlines()
+        lines = [line.strip() for line in lines]
+        reversed_lines = reversed(lines)
+        if tool == 'xcelium':
+            return xcelium_runtimes(reversed_lines)
+        elif tool == 'vcs':
+            return vcs_runtimes(lines)
+        else:
+            raise NotImplementedError(f"{tool} is unsupported for run times extraction.")
+
+
+def vcs_runtimes(reversed_lines: List) -> Tuple[int, int]:
```

Comment:
IMO, this (and the xcelium function) do not need to know or assume the ordering of the `lines` list. 
```suggestion
def vcs_runtimes(lines: List) -> Tuple[int, int]:
```

Reply:
Sounds good, thanks for the suggestion.